superset/models/dashboard.py

```diff
@@ -332,8 +336,8 @@ def alter_positions(
             new_slc_id = Slice.import_obj(slc, remote_slc, import_time=import_time)
             old_to_new_slc_id_dict[slc.id] = new_slc_id
             # update json metadata that deals with slice ids
-            new_slc_id_str = "{}".format(new_slc_id)
-            old_slc_id_str = "{}".format(slc.id)
+            new_slc_id_str = str(new_slc_id)
```

Comment:
Cruft, previous syntax was a safe py2/3 way to stringify... NOTE: there are a dozen other instances of this in the code base `git grep '"{}"\.format'`